### PR TITLE
Few fixes for running the in-room-controller app in release mode

### PR DESCRIPTION
**Pull request by paweldomas**

### CODE REVIEW

File: in-room-controller/App.js [hunk 1]
```diff
@@ -55,11 +55,16 @@ export default class App extends React.Component {
...
+            // The side menu for editing the URL is enabled only in the debug mode.
+            if (__DEV__) {
+                return (
+                    <SideMenu menu = { this._renderSettingsMenu() }>
+                        <RemoteControl url = { remoteControlUrl } />
+                    </SideMenu>
+                );
+            }
+
+            return <RemoteControl url = { remoteControlUrl } />;
```
Comment (paweldomas) [inline in-room-controller/App.js]: is there a better way of doing it ? is it possible to wrap into parent component conditionally ?

Comment (paweldomas) [reply in-room-controller/App.js]: okay I googled something, will update the PR

Comment (virtuacoplenny) [reply in-room-controller/App.js]: process.env?

**Comment by paweldomas** [comment]
Force pushed, because there were eslint errors

File: in-room-controller/.eslintrc [hunk 2]
```diff
@@ -4,6 +4,9 @@
         "eslint-config-jitsi/jsdoc",
         "eslint-config-jitsi/react"
     ],
+    "globals": {
+        "__DEV__": true
```
Comment (virtuacoplenny) [inline in-room-controller/.eslintrc]: How would this get set to false?

Comment (paweldomas) [reply in-room-controller/.eslintrc]: oh I merged before seeing your reply. So this is only for the eslint. It is set by react-native to false when running in release mode

Comment (virtuacoplenny) [reply in-room-controller/.eslintrc]: Ah, right, true meaning it's an allowed global.

**Pull request merged**